pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -148,7 +148,7 @@ stdenv.mkDerivation {
       }
 
       # Install git-subtree.
-      make -C contrib/subtree install ${lib.optionalString withManual "USE_ASCIIDOCTOR=1 install-doc"}
+      make -C contrib/subtree install ${lib.optionalString withManual "USE_ASCIIDOC=1 install-doc"}
```

Comment:
Nitpick but I think we should drop `USE_ASCIIDOC` for simplicity. I don't think it has any effect (or am I missing something?):
```console
$ git show --oneline --no-patch HEAD
773e25afc4 (HEAD -> master, tag: v2.30.1) Git 2.30.1
$ git grep USE_ASCIIDOC | grep -v USE_ASCIIDOCTOR
```

Looking at https://github.com/NixOS/nixpkgs/commit/bbf96d898b96ea2d1ff19a09e545499e4f32d8d0 it might also be possible to drop `docbook_xsl_ns` / to simply revert that commit.

Other than that the Git change LGTM.